examples/distributedLock/UnLockApplication/package.json

```diff
@@ -1,14 +1,14 @@
 {
-    "name": "dapr-example-invocation",
```

Comment:
The directory name also has to change from `UnLockApplication` to `UnlockApplication` as discussed. Should it be `TryUnlock` for consistency with the other example? (Or that should be changed to `LockApplication`?

Reply:
I would like to keep the folder name as "UnlockApplication"